corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/hq.helpers.js

```diff
@@ -75,18 +77,10 @@ hqDefine("hqwebapp/js/bootstrap5/hq.helpers", [
             if (opts) {
                 options = _.extend(options, opts);
             }
-            if (!$link.data('content')) {
```

Comment:
Tested locally that templates missing either a title or content look the same as they do on production.

Reply:
That makes sense given this line in B5 popover docs too:
> Zero-length `title` and `content` values will never show a popover.

Reply:
Yes, tested on staging. Good point about the style guide, I'll take a look at that.

Reply:
Style guide updates: https://github.com/dimagi/commcare-hq/pull/34324